src/PageHeader/PageHeader.features.stories.tsx

```diff
@@ -113,32 +92,34 @@ export const WithActions = () => (
     <PageHeader>
       <PageHeader.TitleArea>
         <PageHeader.Title>Title</PageHeader.Title>
-        <PageHeader.Actions>
-          <IconButton aria-label="Workflows" icon={WorkflowIcon} />
-          <IconButton aria-label="Insights" icon={GraphIcon} />
-          <Button variant="primary" trailingVisual={TriangleDownIcon}>
-            Add Item
-          </Button>
-          <IconButton aria-label="Settings" icon={GearIcon} />
-        </PageHeader.Actions>
       </PageHeader.TitleArea>
+      <PageHeader.Actions>
+        <IconButton aria-label="Workflows" icon={WorkflowIcon} />
+        <IconButton aria-label="Insights" icon={GraphIcon} />
+        <Button variant="primary" trailingVisual={TriangleDownIcon}>
+          Add Item
+        </Button>
+        <IconButton aria-label="Settings" icon={GearIcon} />
+      </PageHeader.Actions>
     </PageHeader>
   </Box>
 )
 
 export const WithDescriptionSlot = () => (
   <Box sx={{padding: 3}}>
-    <PageHeader.TitleArea>
-      <PageHeader.Title>add-pageheader-docs</PageHeader.Title>
-    </PageHeader.TitleArea>
-    <PageHeader.Description>
-      <Text sx={{fontSize: 1, color: 'fg.muted'}}>
-        <Link href="https://github.com/broccolinisoup" sx={{fontWeight: 'bold'}}>
-          broccolinisoup
-        </Link>{' '}
-        created this branch 5 days ago · 14 commits · updated today
-      </Text>
-    </PageHeader.Description>
+    <PageHeader>
```

Comment:
snapshot changes for this story is expected because not wrapping it with PageHeader is not a correct use case and also caused a lack of space between the grid rows.